ecosystem/sep-0010.md

```diff
@@ -19,56 +19,56 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
 It involves three components:
-- A domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`), referred to as the home domain.
-- An endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the home domain, a sub-domain of the home domain, or any other domain. The domain where the endpoint is hosted is referred to as the web auth domain. 
-- A client who will authenticate using a Stellar account.
+- The **TOML Server**: a server hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`). The ``WEB_AUTH_ENDPOINT`` must be hosted by the **Web Auth Server** as described below.
+- The **Web Auth Server**: a domain hosting an endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the **TOML Server**'s domain, a sub-domain of the **TOML Server**'s domain, or any other domain.
+- The **Authenticating Account**: The public key of the Stellar account being authenticated.
+- The **Client Application**: the application used by the owner of the **Authenticating Account**.
```

Comment:
💡 I think `owner` is okay, but `account holder` is more specific and descriptive, and also a common phrase used in financial products which helps provide more depth here.
```suggestion
- The **Client Application**: the application used by the account holder of the **Authenticating Account**.
```